src/finitio/parser.pegjs

```diff
@@ -105,6 +107,11 @@ unnamed_constraint =
     return { native: e.trim() };
   }
 
+type_parameters =
+  "(" head:var_name tail:(opt_comma spacing var_name)* ")" {
```

Comment:
I'll make that change. Also I'd rather leave casing up to the user, but I have no objections to nailing it down to either camelCase or PascalCase.